### DISCUSSION TITLE
App detects LSPosed

### DISCUSSION THREAD

**Discussion by DecentWoodpecker67** [LSPosed/LSPosed#2055] [Q&A]
I've written a module for an app which employs quite a few security mechanisms. In the newest update of the app they seem to be able to detect whether LSPosed is being loaded with the app or not. The module that is being active does not matter, only that LSPosed is active.

The detection mechanism seems to be implemented in a native library (which is obfuscated as far as I know), so I can't really tell how they would be able to do that. Does somebody have experience with such a scenario? Is there a way to further prevent the app from detecting that LSPosed is being loaded?

**Comment by infinity0** [comment]
Revolut is also affected, details in https://github.com/LSPosed/LSPosed/issues/2686

**Comment by infinity0** [comment]
HSBC UK and China apps are both affected in the same way, also since recently. (Versions before worked fine.) Likely HSBC apps in all countries are affected and the problem is not specific to the UK or China.
